PluginCore/PluginCore/FRService/FRSearch.cs

```diff
@@ -608,6 +609,45 @@ private List<SearchMatch> SearchSource(string src, int startIndex, int startLine
             return results;
         }
 
+        private bool LookupRegex(string ba, ref int i)
+        {
+            if (PluginBase.CurrentProject == null || 
```

Comment:
No, we don't have the file name, that was my first thought in fact, but since modifying FRSearch to pass that info meant modifying code in several places and adding more logic, I decided to take this approach.